qlib/backtest/executor.py

```diff
@@ -495,6 +500,22 @@ def _get_order_iterator(self, trade_decision: BaseTradeDecision) -> List[Order]:
             raise NotImplementedError(f"This type of input is not supported")
         return order_it
 
+    def _update_order_num(self, order):
+        """update date and dealed order num in the day."""
+
+        now_deal_day = order.start_time.floor(freq="D")
+        if self.deal_day is None:
+            self.deal_day = now_deal_day
+        if now_deal_day > self.deal_day:
```

Comment:
I think these lines of code can be merged into a single line.

> if self.deal_day is None or now_deal_day > self.deal_day:

Reply:
include `self.deal_order_num = {"buy": defaultdict(int), "sell": defaultdict(int)}` in `__init__`